recipes/opencensus-ext-azure/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "opencensus-ext-azure" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 39f3ad44b2ca7b1f35ea13729f9cf5cb5db8f96ea1ff4a68caf9c15b26249334
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - opencensus >=0.7.11,<1.0.0
```

They've got very unusual versioning for this monorepo - it doesn't follow the versioning of the parent project and they don't tag versions for the sub-projects 😬 Nevertheless, AFAICT the `1.0.6` version should correspond to the `0.7.12` version of the parent project:
```suggestion
    - opencensus >=0.7.12,<1.0.0
```

---

...which means we'll have to wait for the `opencensus-feedstock` PR to be merged.